README.md

```diff
@@ -2,5 +2,81 @@
 
 Framework for integration of WhirlyGlobe-Maply map SDK using Ordnance Survey mapping API.
 
-[![Circle CI](https://circleci.com/gh/OrdnanceSurvey/os-maply-adapter-ios.svg?style=svg)](https://circleci.com/gh/OrdnanceSurvey/os-maply-adapter-ios)
+Build: [![Circle CI](https://circleci.com/gh/OrdnanceSurvey/os-maply-adapter-ios.svg?style=svg)](https://circleci.com/gh/OrdnanceSurvey/os-maply-adapter-ios) [![Coverage Status](https://coveralls.io/repos/github/OrdnanceSurvey/os-maply-adapter-ios/badge.svg?branch=master)](https://coveralls.io/github/OrdnanceSurvey/os-maply-adapter-ios?branch=master)
 
+### Getting started
+
+At the currently time of writing, WG-Maply does not yet support Carthage (see [Add Carthage support](https://github.com/mousebird/WhirlyGlobe/issues/542) issue in main WG-Maply repo). For this reason, manual building of the framework is required. 
+
+Steps to do this are outlined below:
+
+1. Clone this repo
+2. Carthage update (this brings in OSTransformation framework): `$ carthage update --platform iOS --no-use-binaries `
+3. Submodule init (this brings in WG-Maply): `$ git submodule update --init`
```

Comment:
If this was `git submodule update --init --recursive` you probably wouldn't need steps 4 and 5. Your submodule reference should already be the 2.4.1_beta3 tag anyway.